languages.toml

```diff
@@ -2231,3 +2231,28 @@ indent = { tab-width = 8, unit = "        " }
 [[grammar]]
 name = "nasm"
 source = { git = "https://github.com/naclsn/tree-sitter-nasm", rev = "a0db15db6fcfb1bf2cc8702500e55e558825c48b" }
+
+[[language]]
+name = "nim"
+scope = "source.nim"
+injection-regex = "nim"
+file-types = ["nim", "nims", "nimble"]
+shebangs = []
+roots = ["*.nimble"]
```

Comment:
We don't currently support globs or regexes for roots. We might need something more powerful for Nim: from what I can see, Nim uses `<projectname>.nimble` for the manifest?

For now you can leave this as `roots = []` and Helix will use the `.git` directory as the root.